### ISSUE TITLE
Publish to Maven Central

### ISSUE THREAD

**Issue by lolgab** [bblfish/banana-rdf#7]
Until this new version of banana-rdf is unified with the mainline, it would be great if we could publish it to Maven central.
~The group id can be `com.github.bblfish` which is very easy to request to Sonatype.~
EDIT: I just noticed that you already own the groupid.

**Comment by bblfish** [comment]
yes, feel free to update the documentation to make it easier to understand. 

(I am just working on re-implementing Plantain as a pure scala version but with indexes backwards....)
